content/sidebar/student_psy_recommend.md

```diff
@@ -0,0 +1,10 @@
+title: Психологи рекомендуют
+slug: links_student
```

Comment:
`slug` names should be _unique_ across the site and not copypasted. Change it to something more meaningful.